imagetest/test_suites/network/alias_ip_test.go

```diff
@@ -0,0 +1,134 @@
+package network
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	markerFile = "/boot-marker"
+)
+
+func TestAliasAfterReboot(t *testing.T) {
+	_, err := os.Stat(markerFile)
+	if os.IsNotExist(err) {
+		// first boot
+		if _, err := os.Create(markerFile); err != nil {
+			t.Fatalf("failed creating marker file: %v", err)
+		}
+		if err := verifyIPAliases(); err != nil {
```

Comment:
to test the aliases are right on the first boot, i think we should extract that to a second test function TestAliasOnBoot or something. marker-file reboots must always FAIL their first boot.